src/build.py

```diff
@@ -788,23 +791,37 @@ def CMakeCommandBase():
     command.append('-DCMAKE_EXPORT_COMPILE_COMMANDS=ON')
     command.append('-DCMAKE_BUILD_TYPE=Release')
     if IsMac():
-        # Target macOS Seirra (10.12)
-        command.append('-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12')
+        command.append('-DCMAKE_OSX_DEPLOYMENT_TARGET=%s' % OSX_TARGET_VERSION)
     elif IsWindows():
         # CMake's usual logic fails to find LUCI's git on Windows
         git_exe = proc.Which('git')
         command.append('-DGIT_EXECUTABLE=%s' % git_exe)
     return command
 
 
-def CMakeCommandNative(args):
+def CMakeCommandNative(args, build_dir):
     command = CMakeCommandBase()
     command.append('-DCMAKE_INSTALL_PREFIX=%s' % GetInstallDir())
     if IsLinux() and host_toolchains.ShouldUseSysroot():
         command.append('-DCMAKE_SYSROOT=%s' % GetPrebuilt(LINUX_SYSROOT))
         command.append('-DCMAKE_EXE_LINKER_FLAGS=-static-libstdc++')
         command.append('-DCMAKE_SHARED_LINKER_FLAGS=-static-libstdc++')
 
+    elif IsMac() and host_toolchains.ShouldUseSysroot():
+        # Create relpath symlink.
+        print("Creating symlink to %s" % os.path.join(build_dir, 'xcode_sdk'))
+        symlink_dest = os.path.join(build_dir, 'xcode_sdk')
+        if not os.path.exists(symlink_dest):
+            print("creating symlink %s", symlink_dest)
+            os.symlink('/Library/Developer/CommandLineTools/SDKs',
```

Comment:
can we move this hardcoded path up next to OSX_TARGET_VERSION?

Reply:
OK